src/components/HomeSectionClients/index.tsx

```diff
@@ -9,235 +9,228 @@ import {
   FramerLogo,
   Mark,
   LogoText,
-  HomeLeftRightWrapper,
   CommonLogo,
+  TagContainer,
+  CasesItemsContainer,
 } from './styled';
 
 const ClientsSection = () => (
   <Section title="Cases" sectionKind={SectionKind.VERTICAL}>
-    <HomeLeftRightWrapper
-      left={<GoogleLogo />}
-      right={
-        <Content>
+    <CasesItemsContainer>
+      <Content>
+        <LogoText>
+          <GoogleLogo />
           <p>
             We worked with Google on educating developers about web performance.
             This targets the long-term goal of making the web faster.
           </p>
```

Comment:
This text about Google is actually a beginning of the project description, not a company description.

Meaning: It should live under the `<LogoText />` component, not inside it.